internal/cmd/server/flags.go

```diff
@@ -133,11 +133,15 @@ func GetTokenFlag(
 
 // Names of command line flags:
 const (
-	backendTypeFlagName      = "backend-type"
-	backendTokenFlagName     = "backend-token"
-	backendTokenFileFlagName = "backend-token-file"
-	backendURLFlagName       = "backend-url"
-	cloudIDFlagName          = "cloud-id"
-	extensionsFlagName       = "extensions"
-	externalAddressFlagName  = "external-address"
+	backendTypeFlagName       = "backend-type"
+	backendTokenFlagName      = "backend-token"
+	backendTokenFileFlagName  = "backend-token-file"
+	backendURLFlagName        = "backend-url"
+	cloudIDFlagName           = "cloud-id"
+	extensionsFlagName        = "extensions"
+	externalAddressFlagName   = "external-address"
+	namespace                 = "namespace"
```

Comment:
To be consistent with other constants this should be `namespaceFlagName`. Same for the other constants below: should be `...FlagName` for consistency.

Reply:
addressed.